pallets/providers/src/utils.rs

```diff
@@ -841,6 +848,7 @@ impl<T: Config> From<MainStorageProvider<T>> for BackupStorageProvider<T> {
             owner_account: msp.owner_account,
             payment_account: msp.payment_account,
             reputation_weight: T::StartingReputationWeight::get(),
+            sign_up_block: frame_system::Pallet::<T>::block_number(),
```

Comment:
Continuing with having the `sign_up_block` for MSPs as well, this should be `msp.sign_up_block` instead of the current block.

Reply:
You're right on that one. Originally I wasn't going to add `sign_up_block` to MSPs at all, but then I added it, so yes.